### DISCUSSION TITLE
Incorrect website link for pre-training model

### DISCUSSION THREAD

**Discussion by ThomaswellY** [open-mmlab/mmpose#2183] [General]
I have been using mmpose 1.X recently, and wanna download the pretrained model for subsequent training task, and find the website link in https://mmpose.readthedocs.io/en/1.x/model_zoo/body_2d_keypoint.html is incorrect. Can anyone offer any suggestions? Thanks alot in advance !

**Comment by Tau-J** [comment]
Thanks for your feedback. We will fix it today.

**Comment by Ben-Louis** [comment]
Hi @ThomaswellY, the `1.x` branch has been switched to the `main` branch. Please refer to https://mmpose.readthedocs.io/en/latest/model_zoo/body_2d_keypoint.html for the available models

**Reply by ThomaswellY** [reply]
thanks, i clicked the wrong link, the model can still be download at "ckpt"

**Reply by Tau-J** [reply]
> thanks, i clicked the wrong link, the model can still be download at "ckpt"

Hi, the link has been fixed, thanks again for your feedback :)
